FindOpenStudioSDK.cmake

```diff
@@ -110,13 +110,20 @@ else()
             message(STATUS "Download OpenStudio SDK is a success, from: ${OPENSTUDIO_URL}")
             break()
           else()
-            message(FATAL_ERROR
+            set(ERROR_MSG
               "Download seemed to have worked, but archive md5sum HASH mismatch\n"
               "     for file: ${OPENSTUDIO_ARCHIVE_DIR}/${OPENSTUDIO_ARCHIVE_NAME}\n"
               "     from URL: ${OPENSTUDIO_URL}\n"
               "       expected hash: [${OPENSTUDIO_EXPECTED_HASH}]\n"
               "         actual hash: [${OPENSTUDIO_HASH}]\n"
             )
+            # If official => FATAL_ERROR. If CI build, it's possibe the package is overriden by nightly build if the SHA of develop is the same as the
```

Comment:
@jmarrec seems like we should just prevent the nightly builds from overwriting packages? can we request that nightly builds use a different naming convention?  probably just use the date instead of the sha?